FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/FlowcryptApiRepository.kt

```diff
@@ -117,27 +148,9 @@ class FlowcryptApiRepository : ApiRepository {
         )
       }
 
+      val apiService = ApiHelper.getInstance(context).retrofit.create(ApiService::class.java)
       val result = getResult(requestCode = requestCode) { apiService.getPub(identData) }
```

Comment:
`apiService` is a generic name, and `getPub` is also generic - it's not clear where the public key is being fetched from.
`attesterGetPub` or `getPubFromAttester` would be better.